fe/src/components/community/CommunityDetail.tsx

```
@@ -0,0 +1,565 @@
+            <div
+              dangerouslySetInnerHTML={{
+                __html: post.content.replace(/\n/g, "<br>"),
+              }}
+            />
+          ) : (
+            <>
+              <p className="mb-3">
+                학교 밖 생활을 시작한 지 얼마 되지 않아 마음이 복잡했지만, 이번
+                달 &apos;유스보이스&apos;라는 커뮤니티의 월간 소모임에 처음
+                참여하면서 생각이 조금 정리되었다. 모임에서 만난 사람들은 각자
+                다른 이유로 학교 밖을 선택했지만, 공통점은 자기 자리에서 최선을
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)